AppCenterPush/AppCenterPush/MSPush.m

```diff
@@ -118,6 +117,15 @@ + (void *)userNotificationCenterDelegateContext {
 
 #endif
 
+#pragma mark - MSUserIdContextDelegate
+
+- (void)userIdContext:(MSUserIdContext *)__unused userIdContext didUpdateUserId:(NSString *)userId {
+  NSString *pushTokenCopy = self.pushToken;
```

Comment:
Why do you need a copy? `pushToken` will be copied in `[self sendPushToken:pushToken withUserId:userId]`. There is nothing other than just passing the value to a method.